src/cloud.h

```diff
@@ -36,3 +41,4 @@ int cloud_publish_data(const char *id, uint8_t sensor_id,
 		       const knot_value_type *value,
 		       uint8_t kval_len);
 int cloud_unregister_device(const char *id);
+int cloud_register_device(const char *id, const char *name);
```

Comment:
Put before cloud_unregister_device, to keep the order followed in cloud.c

Reply:
Done